root/components/relationship-editor.tt

```diff
@@ -47,11 +47,31 @@
     </div>
     <div>
       <label>
-        <input type="checkbox" data-bind="checked: $parent.changeAllRelationshipCredits" />
-        <span data-bind="html: MB.i18n.l('Change all relationship credits for {entity} on the page.', {entity: entity.html({target: '_blank'})})">
+        <input type="checkbox" data-bind="checked: $parent.changeOtherRelationshipCredits" />
+        <span data-bind="html: MB.i18n.l('Change credits to {entity} for other relationships on the page.', {entity: entity.html({target: '_blank'})})">
         </span>
       </label>
     </div>
+    <div data-bind="visible: $parent.changeOtherRelationshipCredits">
+      <div>
+        <label>
+          <input type="radio" value="all" data-bind="checked: $parent.selectedRelationshipCredits" />
+          [% l('All relationships.'); %]
+        </label>
+      </div>
+      <div>
+        <label>
+          <input type="radio" value="same-entity-types" data-bind="checked: $parent.selectedRelationshipCredits" />
+          [% l('Only relationships with the same entity types.'); %]
+        </label>
+      </div>
+      <div>
+        <label>
+          <input type="radio" value="same-relationship-type" data-bind="checked: $parent.selectedRelationshipCredits" />
+          [% l('Only relationships of the same type.'); %]
```

Comment:
It might benefit users to make these messages a bit more specific too, like "Only relationships to {entity_type} entities" and "Only relationships of type {relationship_type}" (especially since the user can change the relationship type, so it's not clear which one it uses).